script/testing/junit/src/GenerateTrace.java

```diff
@@ -61,25 +59,16 @@ public static void main(String[] args) throws Throwable {
                 String[] lines = line.split(",");
                 writeToFile(writer, QUERY_I_NOSORT);
                 writeToFile(writer, line);
-                writeToFile(writer, SEPARATION);
+                writeToFile(writer, constants.SEPARATION);
                 ResultSet rs = statement.getResultSet();
                 List<String> res = mog.processResults(rs);
                 // compute the hash
-                MessageDigest md;
-                try {
-                    md = MessageDigest.getInstance("MD5");
-                } catch (NoSuchAlgorithmException e) {
-                    throw new RuntimeException("no Alg", e);
-                }
-                String resultString = String.join("\n", res) + "\n";
-                md.update(resultString.getBytes());
-                byte[] byteArr = md.digest();
-                String hash = MogUtil.bytesToHex(byteArr).toLowerCase();
+                String hash = TestUtility.getHashFromDb(res);
```

Comment:
Something I keep running into is that if the number of decimal points are different in postgres and terrier I'll get the wrong hash. May be worthwhile to include it in the README somewhere (not sure what the fix is though, maybe if possible to change the precision of the postgres responses. Or maybe in the future we can add some rounding before hashing).